packages/documentation/src/pages/platform/site-structure/application-structure.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Page and application Javascript structure
+tags: page, bundle, react, javascript, structure, webpack, site-wide
```

Comment:
you can leave out words that are in the title unless you want them to show up as matching tags

![Screen Shot 2019-05-23 at 08 50 42](https://user-images.githubusercontent.com/4998130/58262633-de6efe00-7d37-11e9-8b22-f447dff51a0a.png)